### ISSUE TITLE
Overflow statistic inaccurate

### ISSUE THREAD

**Issue by agorrod** [wiredtiger/wiredtiger#1703]
I've created a row store with overflow records and statistics are reporting that we don't have overflow items. A script to generate the table and gather statistics follows.

```
cd build_posix
../configure > /dev/null && make -j 12 > /dev/null

DBDIR=./WT_TEST
WT=./wt

rm -rf $DBDIR
mkdir -p $DBDIR

$WT -h $DBDIR create  -c "key_format=S,value_format=S,internal_page_max=4KB,leaf_page_max=4KB" table:test

value=`seq -s ' ' 1 10000`
$WT -h $DBDIR write table:test fill_2__b_27 "$value"
$WT -h $DBDIR write table:test fill_2__b_28 "$value"
echo "Reporting table statistics for overflow items"
$WT -h $DBDIR stat table:test | grep overflow
echo "Reporting overflow information from block dump"
$WT -h $DBDIR verify -d dump_blocks table:test | grep ovf
```

Generates the following:

```
Reporting table statistics for overflow items
btree: overflow pages=0
cache: overflow values cached in memory=0
cache: overflow pages read into cache=0
reconciliation: internal-page overflow keys=0
reconciliation: leaf-page overflow keys=0
reconciliation: overflow values written=0
Reporting overflow information from block dump
	value/ovfl: len 7, ovfl [4096-53248, 49152, 2123498883]
	value/ovfl: len 7, ovfl [65536-114688, 49152, 1832075221]
```

**Comment by agorrod** [comment]
@keithbostic I had a bit of a look at this, and the code in ```src/btree/bt_stat.c``` looks wrong to me. It's expecting to see ```WT_PAGE_OVFL``` type pages from a ```__wt_tree_walk```. I don't think we'll ever see those - a walk only traverses pages, it won't go looking for overflow pages referenced by keys/values.

I haven't pursued this further, because the only way I can see to get the information is to unpack the cell of each entry in the tree and look at the resulting overflow field. Overflow statistics are very interesting - so I think it's worth the overhead, but I hoped you'd have a cleaner/cheaper solution.

**Comment by keithbostic** [comment]
@agorrod, yes, your analysis is exactly correct.

I agree the only way to do this is to unpack the cell and look at the key/value items.

**Comment by agorrod** [comment]
This ticket overlaps with #1520

**Comment by keithbostic** [comment]
Closed with the merge of #1711.
